sdk/iot/Azure.Iot.Hub.Service/tests/IotHubServiceTestEnvironment.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    // This class contains the configurations required to be set to run tests against the CI pipeline.
+    public class IotHubServiceTestEnvironment : TestEnvironment
+    {
+        public IotHubServiceTestEnvironment()
+            : base("iot")
```

Comment:
This should be TestSettings.IotHubServiceEnvironmentVariablesPrefix

Reply:
Tracked here: https://msazure.visualstudio.com/One/_workitems/edit/7463729